e2e/test/scenarios/joins/joins.cy.spec.js

```diff
@@ -292,11 +292,11 @@ describe("scenarios > question > joined questions", () => {
     // Test LHS column infers RHS column's temporal unit
 
     cy.findByLabelText("Left column").click();
-    popover().findByText("by month").click({ force: true });
-    popover().last().findByText("Week").click();
+    popover().findByText("Created At").click();
```

Comment:
I believe that was an actual bug. At least the test didn't do what the comment says "Test LHS column infers RHS column's temporal unit". Previously the temporal unit of the RHS column was always overwritten by the unit from the LHS column. Now the unit of the last selected column "wins".